CMakePresets.json

```diff
@@ -0,0 +1,303 @@
+{
+    "version": 6,
+    "cmakeMinimumRequired": {
+      "major": 3,
+      "minor": 14,
+      "patch": 0
+    },
+    "configurePresets": [
+      {
+        "name": "common",
+        "hidden": true,
+        "cacheVariables": {
+          "FREECAD_USE_PYBIND11": {
+            "type": "BOOL",
+            "value": "ON"
+          }
+        }
+      },
+      {
+        "name": "debug",
+        "displayName": "Debug",
+        "description": "Default debug profile",
+        "binaryDir": "${sourceDir}/build/debug",
+        "inherits": [
+          "common"
+        ],
+        "cacheVariables": {
+          "CMAKE_BUILD_TYPE": {
+            "type": "STRING",
+            "value": "Debug"
+          }
+        }
+      },
+      {
+        "name": "release",
+        "displayName": "Release",
+        "description": "Default release profile",
+        "binaryDir": "${sourceDir}/build/release",
+        "inherits": [
+          "common"
+        ],
+        "cacheVariables": {
+          "CMAKE_BUILD_TYPE": {
+            "type": "STRING",
+            "value": "Release"
+          }
+        }
+      },
+      {
+        "name": "conda",
+        "hidden": true,
+        "generator": "Ninja",
+        "cacheVariables": {
+          "Boost_NO_BOOST_CMAKE": {
+            "type": "BOOL",
+            "value": "ON"
+          },
+          "BUILD_DYNAMIC_LINK_PYTHON": {
+            "type": "BOOL",
+            "value": "ON"
+          },
+          "BUILD_FEM_NETGEN": {
+            "type": "BOOL",
+            "value": "ON"
+          },
+          "BUILD_FLAT_MESH": {
+            "type": "BOOL",
+            "value": "ON"
+          },
+          "BUILD_REVERSEENGINEERING": {
+            "type": "BOOL",
+            "value": "ON"
+          },
```

Comment:
this defaults to ON so it doesn't need to be specified, furthermore this one might ge axed in the future since it's unmaintained so probably best not to add more places we're we'll have to change defaults when that happens